ajax/libs/beepjs/package.json

```diff
@@ -9,6 +9,7 @@
     "email": "neuman@ashmoremusic.com",
     "url": "https://github.com/luciferous"
   },
+  "license": "BSD 2-Clause",
```

Comment:
@yangwanting How do you know that it's BSD-2-Clause license? 
BTW,You should use `BSD-2-Clause` not `BSD 2-Clause`.